website/docs/r/kusto_cluster.html.markdown

```diff
@@ -70,6 +70,11 @@ The following arguments are supported:
 
 * `virtual_network_configuration` - (Optional) A `virtual_network_configuration` block as defined below. Changing this forces a new resource to be created.
 
+~> **NOTE:** Removing `virtual_network_configuration` from template will set the value of `public_network_access_enabled` field to be `false`.
+
+~> **NOTE:** Currently removing `virtual_network_configuration` sets the `virtual_network_configuration` to `Disabled` state. But any changes to `virtual_network_configuration` in `Disabled` state forces a new resource to be created.
```

Comment:
Can we remove this? Since that won't be possible via Terraform because the `virtual_network_configuration` would have to be in the config file to change it at which point the `state` would be set to `Enabled`?
```suggestion
```